grails-app/i18n/messages.properties

```diff
@@ -3358,6 +3358,12 @@ react.stockMovement.deleted.success.message.label=Stock Movement has been delete
 react.stockMovement.inbound.fetching.error=Unable to fetch inbounds movements
 react.stockMovement.outbound.fetching.error=Unable to fetch outbound movements
 react.stockMovement.inbound.filters.receiptStatus.label=Receipt Status
+react.stockMovement.filters.shipmentType.label=Shipment type
```

Comment:
do we need a separate message for the filter. i think we should just use the same label. If one changes, the other one needs to change. 
```
react.stockMovement.shipmentType.label
```